bootstrap.sh

```diff
@@ -3,21 +3,31 @@ set -o errexit -o pipefail -o noclobber -o nounset
 
 getopt --test && echo "'getopt --test' failed in this environment." && exit 1
 
-LONGOPTS=leap:,cdt:,help
-OPTIONS=l:c:h
+# Set SDIR to the path containing this script. Even if it's a symlink.
+SDIR=$(dirname $(readlink -f "$0"))
```

Comment:
```suggestion
SDIR=$(dirname "$(readlink -f "$0")")
```

Reply:
This is a very good suggestion! Fixed in e1a49e9